eBookReader/chat.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Q&A</title>
+	<link rel="stylesheet" type="text/css" href="styleIndex.css"/>
+	<link rel="stylesheet" type="text/css" href="debugStyle.css"/>
+</head>
+<body>
+<header>
+	<p class = "name">John Doe</p>
+	<p class = "userStats">
+		Contribution points: 91
+		<a href="#" class="iconButton">?</a>
+	</p>
+</header>
+
+<h2>According to the author, why do mass extinctions happen?</h2>
+
+<div class = "chatPanel">
+	<div class = "chatMessage">I believe that on page 3, the author clearly states that extinctions happen for three reasons.</div>
+	<div class = "contributePanel">
+		<img class = "centerBadge answerBadge" src = "orange triangle.png" alt = "Caution Triangle" height = "50" width = "50">
+		<a class = "button contributeButton" href="#" >Contribute</a>
+	</div>
+</div>
+
+<p class = "timeStamp chat">Anthony Stark | 2 hours ago</p>
```

Comment:
You want these timestamps to be inside the chatPanel div. Think of each chatPanel as a reusable block, that we can programmatically instantiate whenever we have a new chat message. This timestamp is part of the panel--semantically, therefore, it makes more sense for it to contain the timestamp as well.